docs/user-guide/rules.md

```diff
@@ -240,6 +240,7 @@ Here are all the rules within stylelint, grouped by the [*thing*](http://apps.wo
 - [`no-browser-hacks`](../../src/rules/no-browser-hacks/README.md): Disallow browser hacks that are irrelevant to the browsers you are targeting.
 - [`no-descending-specificity`](../../src/rules/no-descending-specificity/README.md): Disallow selectors of lower specificity from coming after overriding selectors of higher specificity.
 - [`no-duplicate-selectors`](../../src/rules/no-duplicate-selectors/README.md): Disallow duplicate selectors.
+- [`no-empty-source`](../../src/rules/no-empty-source/README.md): Disallow empty source.
```

Comment:
"Whitespace" is a different kind of word, which can be used in a way such that it can refer to multiple things without taking a plural form. Like "butter" and "cattle". You'd say "Look at all that butter", not "Look at all those butters". "Whitespace" is weird because it can actually be used both ways: "Delete that whitespace" or "Delete those whitespaces" — but "source" always takes an "s" in plural.